resources/provider/migrations/postgres/5_v0.4.1.up.sql

```diff
@@ -0,0 +1,37 @@
+-- Autogenerated by migration tool on 2022-02-04 16:21:37
```

Comment:
I think it is better to store all migrations in unreleased.up.sql

Reply:
unreleased won't work, as it can't be tested by migrations, and the tester will fail your test (as of SDK v0.7.6+)